pkg/pipelines/pipelines.go

```diff
@@ -0,0 +1,100 @@
+package pipelines
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/openshift-pipelines/release-tests/pkg/client"
+	"github.com/openshift-pipelines/release-tests/pkg/helper"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	tb "github.com/tektoncd/pipeline/test/builder"
+)
+
+var (
+	createFileTaskName        = "create-file"
+	readFileTaskName          = "check-stuff-file-exists"
+	tePipelineName            = "output-pipeline"
+	tePipelineGitResourceName = "skaffold-git"
+)
+
+func getGitResourceForOutputPipeline(rname string, namespace string) *v1alpha1.PipelineResource {
```

Comment:
You use different naming conventions in file `pipeline_status.go`. I personally prefer `new` over `get`